templates/unicorn.rb

```diff
@@ -1,30 +0,0 @@
-# https://devcenter.heroku.com/articles/rails-unicorn
-
-worker_processes (ENV["UNICORN_WORKERS"] || 3).to_i
-timeout (ENV["UNICORN_TIMEOUT"] || 15).to_i
```

Comment:
With `UNICORN_TIMEOUT` gone and no timeout built into Puma, should we increase our default `RACK_TIMEOUT` to 20?

https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#timeout